### PR TITLE
Create clio.yml

### PR DESCRIPTION
**Pull request by garritfra**

### CODE REVIEW

File: _data/projects/clio.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+name: Clio
+desc: Clio is a pure functional lazy-evaluated programming language targeting decentralized and distributed systems.
+site: https://clio-lang.org
+tags:
+  - language
+  - programming
+  - distributed
+  - functional
+  - fp
```
Comment (ritwik12) [inline _data/projects/clio.yml:10]: @garritfra Do you have any programming languages to add as tag?  It will be better

Comment (garritfra) [reply _data/projects/clio.yml:10]: Added with f045492

**Comment by garritfra** [comment]
@ritwik12  I have added the tags

**Comment by garritfra** [comment]
@ritwik12 dont mean to bother, but can you review this PR? Thanks! :)

**Review by shiftkey — APPROVED** [review]
@garritfra thanks for the contribution!

**Comment by ritwik12** [comment]
@garritfra Sorry but I guess I missed your ping as I couldn't read my mails from few days. Thanks @shiftkey

**Pull request merged**